packages/core/src/__tests__/document.test.ts

```diff
@@ -412,7 +453,7 @@ describe('Document', () => {
 
       try {
         await Document.load(doc.id, doctypeHandler, dispatcher, pinStore, context, { sync: false })
-        throw new Error('Should not be able to assign a schema to a document that does not conform')
+        throw new Error("Should not be able to load a document that doesn't conform to its schema")
```

Comment:
`fail('Should not be able to load a document that doesn't conform to its schema')`